qiskit/pulse/instructions/__init__.py

```diff
@@ -13,10 +13,12 @@
 # that they have been altered from the originals.
 
 """The ``instruction`` module holds the various ``Instruction`` s which are supported by
-Qiskit Pulse. Instructions each contain operands unique to that instruction, but all have an
-implicit or explicit duration, and a channel on which it is played.
+Qiskit Pulse. Instructions accept a list of operands unique to instructions of that type. Every
+instruction includes at least one :py:class:`~qiskit.pulse.channels.Channel` as an operand
+specifying where the instruction will be applied, and every instruction has either an explict
+duration as one of its operands or an implicit duration of ``0``.
```

Comment:
This is not completely True, every Instruction has a duration, but it is not necessarily supplied, nor 0, eg., a Play derives its duration from the Pulse. The statement "every Instruction has a duration" is True.

Reply:
I'd thought of Play as having an explicit duration but you're right, it doesn't really